Makefile

```diff
@@ -137,6 +137,20 @@ start:
 	fi
 	@docker-compose up -d
 
+.PHONY: gitpod-start
+gitpod-start: # start local setup
+gitpod-start:
+	@if [ ! -d "local_configs" ]; then \
+		mkdir -p local_configs && \
+		mkdir -p local_configs/backend && \
+		mkdir -p local_configs/monitor && \
+		mkdir -p local_configs/frontend && \
+		cp -rn backend-services/configs/* local_configs/backend && \
+		cp -rn monitor-services/configs/* local_configs/monitor && \
+		cp other/frontend/configs/nginx-gitpod.conf local_configs/frontend/nginx.conf; \
```

Comment:
The correct command is:
```
mkdir -p local_configs && \
mkdir -p local_configs/backend && \
mkdir -p local_configs/monitor && \
mkdir -p local_configs/frontend && \
cp -rn backend-services/configs/* local_configs/backend && \
cp backend-services/configs/redis.conf local_configs/backend/redis.conf && \
cp -rn monitor-services/configs/* local_configs/monitor && \
cp -rn other/frontend/configs/* local_configs/frontend
```

Reply:
so you simply need to add the redis conf so that it works as expected. Otherwise the PR is fine.